types/mempool/sender_nonce.go

```diff
@@ -26,15 +26,29 @@ var (
 //
 // Note that PrepareProposal could choose to stop iteration before reaching the end if maxBytes is reached.
 type senderNonceMempool struct {
-	senders map[string]*skiplist.SkipList
-	rnd     *rand.Rand
+	senders    map[string]*skiplist.SkipList
+	rnd        *rand.Rand
+	maxTx      int
+	unbounded  bool
```

Comment:
I think if maxtx was something like 1m it would equate the same?

Reply:
Yes, I was just thinking of being explicit but that works too